actors/migration/top.go

```diff
@@ -129,60 +159,62 @@ func MigrateStateTree(ctx context.Context, store cbor.IpldStore, stateRootIn cid
 		return cid.Undef, err
 	}
 
-	// Extra setup
-	// miner
-	var p phoenix
-	if err := p.load(ctx, actorsIn); err != nil {
-		return cid.Undef, err
-	}
+	// Extra actor setup
 	// power
 	pm := migrations[builtin0.StoragePowerActorCodeID].StateMigration.(*powerMigrator)
 	pm.actorsIn = actorsIn
 
+	// Setup synchronization
+	errCh := make(chan error)
+	transferInCh := make(chan big.Int)
+	transferOutCh := make(chan big.Int)
```

Comment:
Many channels is really complicated. I can't quite follow what they're all doing, but prefer a single channel with a union-style struct value type over many channels with different value types.